eclair-core/src/test/scala/fr/acinq/eclair/channel/states/g/NegotiatingStateSpec.scala

```diff
@@ -139,6 +153,10 @@ class NegotiatingStateSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike
     testFeeConverge(f)
   }
 
+  test("recv ClosingSigned (theirCloseFee == ourCloseFee) (fee 1, upfront shutdown script)", Tag("fee1"), Tag(StateTestsTags.OptionUpfrontShutdownScript)) { f =>
```

Comment:
I think the "fee1" tag doesn't exist, it's the default behavior.
```suggestion
  test("recv ClosingSigned (theirCloseFee == ourCloseFee) (fee 1, upfront shutdown script)", Tag(StateTestsTags.OptionUpfrontShutdownScript)) { f =>
```